src/olympia/pages/templates/pages/shield_study_6.html

```diff
@@ -0,0 +1,63 @@
+        </div>
+        <div class="content"><p>Mozilla takes your security and privacy seriously and wants to know if there are ways we can keep you even safer on the web. Participating in this study will help us find out!</p>
+<h2 id="what-will-happen-next">What will happen next</h2>
+<p><strong>If you agree</strong> to participate:</p>
+<ol>
+<li>We will install an add-on that provides additional security-related features while browsing the web.</li>
+<li>Continue using Firefox as you normally would. You may notice not notice any changes immediately.</li>
+<li>After 14 days, the study will end on its own. Your Firefox will go back to normal.</li>
+<li>We will ask you to fill out a quick <strong>optional survey</strong> to tell us about your experience. The survey should take less than 5 minutes to complete.</li>
+</ol>
+<h2 id="leaving-the-study">Leaving the study</h2>
+<p>The study will expire on its own in 14 days.  You may leave the study early.  To do so, follow these steps:</p>
+<ol>
+<li>Type <code>about:addons</code> into the location bar, and press enter.</li>
+<li>Find <code>Security Advisor</code> in the <strong>extensions</strong> list pane.</li>
+<li>Click the <code>remove</code> button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h2 id="your-privacy">Your privacy</h2>
+<h3 id="all-shield-studies">All Shield Studies</h3>
+<p>Every Shield Study collects data about important study events, such as install, uninstall, daily usage, and end of study. Studies also include an optional survey.</p>
+<p>This data is associated with <a href="https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Firefox Telemetry collection</a>.</p>
+<h3 id="this-particular-study">This Particular Study</h3>
+<p>In addition to the data collected by all Shield Studies and your voluntary answers to the followup survey, here are the <strong>key things you should know</strong> about what is happening when you participate in this study:</p>
+<ul>
+<li>We collect basic usage data regarding your interaction with the experimental add-on, such as data when you use a new security feature.</li>
+<li>We collect information about your visits to a few popular sites and if you have a saved login for that site. No login credentials will be tracked or saved.</li>
+<li>As you would expect, we do not collect any information when you are in private browsing mode.</li>
+</ul>
+<p>You can learn more about the data collection for this study <a href="https://github.com/mozilla/security-advisor-shield-study/blob/master/domains.txt">here</a></p>
```

Comment:
Would be really nice to fix this link to say more than "here" for better accessibility.